packages/pigeon/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 4.2.11
+## 4.3.0
 
 * Add documentation comment support for Enum members.
```

Comment:
This just needs to start with `adds` instead of `add`. You can change the version number back to 4.2.11. Sorry for the confusion.